x-pack/test/security_solution_api_integration/test_suites/detections_response/rules_management/rule_read/trial_license_complete_tier/find_rules.ts

```diff
@@ -145,13 +145,17 @@ export default ({ getService }: FtrProviderContext): void => {
       const rule: ReturnType<typeof getSimpleRule> = {
         ...getSimpleRule('rule-1'),
         throttle: '1h', // <-- throttle makes this a scheduled action
+        interval: '50m', // has to be less than "throttle"
```

Comment:
Why does this have to be less than throttle here but equal in other places? Just because we're testing action logic?

Reply:
Actually, you're right. I have re-checked and it looks like we can use `24h` in both places.